ipv-api/src/main/java/uk/gov/di/authentication/ipv/entity/IdentityProgressResponse.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.di.authentication.ipv.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import com.nimbusds.oauth2.sdk.id.State;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.net.URI;
+
+public class IdentityProgressResponse {
+
+    @SerializedName("status")
+    @Expose
+    @Required
+    private IdentityProgressStatus status;
+
+    @SerializedName("client-name")
+    @Expose
+    @Required
+    private String clientName;
+
+    @SerializedName("redirect-uri")
+    @Expose
+    @Required
+    private URI redirectUri;
+
+    @SerializedName("state")
+    @Expose
+    @Required
+    private State state;
+
+    public IdentityProgressResponse() {}
```

Comment:
Why do we need the no-args constructor here?

Reply:
I think this is just me not having that much java experience, and having seen other examples in the code so thinking I needed it 😅